import-automation/progress-dashboard-rest/test/test_import_attempt.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Tests for import_attempt.py.
+"""
+
+
+from context import app
+from app.resource import import_attempt
+from app.service import import_attempt_database_dict
+
+import unittest
+from unittest import mock
+
+
+PARSE_ARGS = 'flask_restful.reqparse.RequestParser.parse_args'
+IMPORT_ATTEMPT_DATABASE = 'app.resource.import_attempt' \
+                          '.import_attempt_database.ImportAttemptDatabase'
+EXAMPLE_ATTEMPT = {
+    'attempt_id': '0', 'branch_name': 'branch',
```

Comment:
Please have one entry per line, easing human comprehension.

Reply:
fixed

Reply:
I do not see the changes, but perhaps they are elsewhere.  I am still learning the GitHub UI.